WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdViewModel.swift

```diff
@@ -71,17 +65,36 @@ final class BlazeEditAdViewModel: ObservableObject {
                                description: description)
     }
 
+    @Published private var selectedSuggestionIndex: Int?
```

Comment:
Nice to see the use of nullable Int here as both a flag to allow / disallow selection, as well as to store the index data.

Just interesting because I ran into a similar situation today and came up [with a similar solution](https://github.com/woocommerce/woocommerce-ios/pull/11642/files#diff-5baafa8e37a66b2435d67e550f40b4da9c56196a227590dba9a94114852de463R19).